novelai-api/api.go

```diff
@@ -254,12 +272,15 @@ func naiApiGenerate(keys *NaiKeys, params NaiGenerateMsg, backend string) (respD
 	if len(*params.Parameters.BadWordsIds) == 0 {
 		params.Parameters.BadWordsIds = nil
 	}
+	if len(*params.Parameters.LogitBiasIds) == 0 {
```

Comment:
In a default case, this will never be true, due to having an array element size of `1`.